bin/beeline

```diff
@@ -23,8 +23,29 @@
 # Enter posix mode for bash
 set -o posix
 
+realpath () {
```

Comment:
I see, I would imagine it could be next to the symlink and symlinked as well... but is the point that even sourcing isn't going to follow the symlink? I thought the point was not that the symlinks don't work as links to files, but that they don't help you find where Spark is installed.